CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
-cmake_minimum_required(VERSION 3.1.0)
+cmake_minimum_required(VERSION 3.16.0)
```

Comment:
This is creating a big mess probably unnecessarily. On Debian 10 cmake is at version 3.13.4 for example.